bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -150,65 +148,76 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
             String contentType, int timeout, String proxyHost, Integer proxyPort, String proxyUser,
             String proxyPassword, String nonProxyHosts) {
 
-        HttpClient client = new HttpClient();
+        startHttpClient(client);
 
         // only configure a proxy if a host is provided
         if (StringUtils.isNotBlank(proxyHost) && proxyPort != null && shouldUseProxy(url, nonProxyHosts)) {
-            client.getHostConfiguration().setProxy(proxyHost, proxyPort);
-            if (StringUtils.isNotBlank(proxyUser)) {
-                client.getState().setProxyCredentials(AuthScope.ANY,
-                        new UsernamePasswordCredentials(proxyUser, proxyPassword));
+            AuthenticationStore authStore = client.getAuthenticationStore();
+            ProxyConfiguration proxyConfig = client.getProxyConfiguration();
+
+            // clear proxy list from previous requests
+            List<Proxy> proxies = proxyConfig.getProxies();
+            if (!proxies.isEmpty()) {
+                proxies.clear();
             }
+
+            HttpProxy proxy = new HttpProxy(proxyHost, proxyPort);
+            proxies.add(proxy);
+
+            authStore.addAuthentication(new BasicAuthentication(proxy.getURI(), null, proxyUser, proxyPassword));
```

Comment:
@kaikreuzer 
No I didn`t test it.
I tried to find a bundle, that is using the proxy support, but I could not find one. I searched through [this list ](https://github.com/eclipse/smarthome/issues/1689#issuecomment-228032703).
I could not find info about "any realm" as well.